modules/core/core-schema/src/main/java/com/enonic/xp/core/impl/schema/mapper/DescriptorKeyDeserializer.java

```diff
@@ -21,15 +21,15 @@ public DescriptorKey deserialize( final JsonParser jsonParser, final Deserializa
         final ObjectMapper mapper = (ObjectMapper) jsonParser.getCodec();
         final JsonNode node = mapper.readTree( jsonParser );
 
-        final String rawValue = node.asText();
-        if ( rawValue.contains( ":" ) )
+        final String trimmedValue = node.asText().trim();
```

Comment:
I see no pint in yaml/json to trim values. it might have had some explanation n XML only